packages/documentation/src/stories/components/avatar/avatar.docs.mdx

```diff
@@ -0,0 +1,38 @@
+import * as AvatarPictureStories from './avatar.stories';
+
+<Meta of={AvatarPictureStories} />
+
+<div className="docs-title">
+  # Avatar
+
+  <nav>
+    <link-design of={JSON.stringify(AvatarPictureStories)}></link-design>
+  </nav>
+</div>
+
+<p className="lead">Display a user's profile picture or their initials if the picture is unavailable.</p>
+
+<Canvas of={AvatarPictureStories.Default} />
+<Controls of={AvatarPictureStories.Default} />
+
+## Properties
+
+The `<post-avatar>` component provides several possibilities to change its appearance. While the only required property is `firstname`, you can choose freely, which of the other properties you want to use. You can even combine them. The component will always try to show a picture, if possible, and fallback to initials, if not.
+Please notice, that if you combine several methods to show a picture, the component will use the first one it can get a picture from.
+
+The properties are used in the order as follows:
+
+1. If you use an `<img>` tag in the default slot (for example to show a self hosted image), this will override the other properties.<br/><br/>
+  <div className="mb-regular alert alert-info alert-md">We do not check if the image can be loaded, so make sure the image is available!</div>
+2. If you use the `userid` property (to show a company internal user picture) and a picture is available, this will override the `email` property, as well as `firstname` and `lastname`.<br/><br/>
+  <div className="mb-regular alert alert-info alert-md">This will only work on `post.ch` urls and only in internal projects.</div>
+3. If you use the `email` property (to show a users picture provided through gravatar.com) and a picture is available, this will override the `firstname` and `lastname` properties.<br/><br/>
+  <div className="mb-regular alert alert-info alert-md">Don't know Gravatar or haven't set up a profile yet, let's change this: [Signup](https://gravatar.com) now!</div>
+4. As an overall fallback the `firstname` and `lastname` properties are used to show at least some initials. And as an addition, `firstname` and `lastname` (if defined) are also used on all the other methods to set the `alt` tag of the image (if not already defined).<br/><br/>
+  <div className="mb-regular alert alert-info alert-md">Because of big differences in the naming convenions around the world, we only set the `firstname` as required property.</div>
+
+## Check the Content Security Policy
+
+If your project includes the <a href="https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy">Content-Security-Policy</a> response header (usually in `index.html`),
+make sure that `default-src` is set to `'self'`. In case you are using the `<post-avatar>` component you need to add [`https://gravatar.com`](https://gravatar.com/) to the `connect-src` in index.html. This is because the component tries to load a profile picture from there, through the provided `email` property.
```

Comment:
```suggestion
make sure that `default-src` is set to `'self'`. The `<post-avatar>` component is capable of loading images from gravatar, therefore you need to add [`https://gravatar.com`](https://gravatar.com/) to the `connect-src` in index.html. This is because the component tries to load a profile picture from there, through the provided `email` property.
```